go.sum

```diff
@@ -24,6 +24,7 @@ code.cloudfoundry.org/clock v1.0.0 h1:kFXWQM4bxYvdBw2X8BbBeXwQNgfoWv1vqAk2ZZyBN2
 code.cloudfoundry.org/clock v1.0.0/go.mod h1:QD9Lzhd/ux6eNQVUDVRJX/RKTigpewimNYBi7ivZKY8=
```

Comment:
This PR changes a lot of lines in go.sum but the plugin itself only reads a file in proc.  Why is go.sum changed so much?  It seems unrelated.

CI dep checs aren't passing.  You'll need to run 'go mod tidy' and commit the changes.

Reply:
You are right. I didn't do anything that would change the dependencies. `go mod tidy` cleaned it up.